### ISSUE TITLE
Empty data message sent on subscription request from Apollo

### ISSUE THREAD

**Issue by xaralis** [datadvance/DjangoChannelsGraphqlWs#4]
When following the docs, empty `data` type message is sent via WS once new subscription request is made as a subscription confirmation. This is clearly unexpected behavior in case `vue-apollo` is used on the frontend. It complains about empty message.

Do you thinks this is invalid implementation on your end or theirs?

**Comment by prokher** [comment]
This is a bug. Initially I implemented the server following [the Apollo GraphQL protocol description](https://github.com/apollographql/subscriptions-transport-ws/blob/master/PROTOCOL.md) which states:

> For each GraphQL operation send with GQL_START, the server will respond with at least one GQL_DATA message.

so I decided to respond with `GQL_DATA` to "confirm" the subscription. Reading Apollo GraphQL server code has revealed that this is unnecessary. The fix will come soon among with the bunch of other stuff.
